config_example.json

```diff
@@ -1293,15 +1293,7 @@
    "websocketOrderbookBufferLimit": 5,
    "baseCurrencies": "USD",
    "currencyPairs": {
-    "requestFormat": {
-     "uppercase": false,
-     "delimiter": "_"
-    },
-    "configFormat": {
-     "uppercase": true,
-     "delimiter": "_"
-    },
-    "useGlobalFormat": true,
+    "useGlobalFormat": false,
```

Comment:
Can remove this too as this will default to false (the less clutter, the better)